src/test/java/uk/gov/hmcts/ccd/domain/service/startevent/DefaultStartEventOperationTest.java

```diff
@@ -100,6 +100,9 @@
                                                                  + "  \"OrgPolicyCaseAssignedRole\": \"[Claimant]\""
                                                                  + "}");
             data.put("OrganisationPolicyField", orgPolicyDefaultValue);
+
+            JsonNode textFieldDefaultValue = MAPPER.readTree("{ \"TextField0\": \"Default text\" }");
+            data.put("TextField0", textFieldDefaultValue);
```

Comment:
This is currently creating an `ObjectNode` with key `TextField0` which contains a subfield also called `TextField0` - for a top level field can use the similar approach as in `CaseService`:
```suggestion
            data.put("TextField0", MAPPER.getNodeFactory().textNode("Default text"));
```
(Same comment applies to other two places in this file)

Reply:
Also worth adding another top level field in here for another field, but don't add it into the existing case data - to make sure that fields which don't exist also work